wagtail/admin/templates/wagtailadmin/shared/datetimepicker_translations.html

```diff
@@ -36,4 +36,14 @@
         ],
     }
     $.datetimepicker.setLocale('wagtail_custom_locale');
+
+    {#
```

Comment:
This comment remains intact in the output, causing a JS error. It should probably use `{% comment %}` / `{% endcomment %}` instead...